src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/ProbabilityTable/Builder/src/CDFs.hpp

```diff
@@ -0,0 +1,18 @@
+template< typename Range,
+          utility::Require< true, utility::is_range, Range > = true >
+Builder& CDFs( Range&& cdfs ){
```

Comment:
I think this file can be removed?

Reply:
No, I'm still using it. Why do you think it can be removed?